src/js/structs/Pod.js

```diff
@@ -138,6 +138,10 @@ module.exports = class Pod extends Service {
     return this._spec;
   }
 
+  getStatusString() {
+    return (this.get('status') || '').toUpperCase();
```

Comment:
I agree that we don't need a utility function for this one, but I wonder why we upper case the status string?